lib/modules/platform/utils/read-only-issue-body.spec.ts

```diff
@@ -14,7 +14,7 @@ describe('modules/platform/utils/read-only-issue-body', () => {
     it('removes all checkbox-related instructions', () => {
       expect(readOnlyIssueBody(issueBody)).toEqual(
         expect.not.stringMatching(
-          /click (?:(?:on |)a|their) checkbox|check the box below/gi
+          /select (?:(?:on |)a|their) checkbox|check the box below/gi
```

Comment:
This line needs to be changed as well, I think.

Reply:
this needs to be more fixed to match the changed wording in file below